src/ocm/components/clusterDetail/ProgressBarAlerts.tsx

```diff
@@ -76,9 +79,9 @@ export const HostInstallationWarning: React.FC<installationProgressWarningProps>
         </RenderIf>
         <RenderIf condition={failedOperators?.length > 0}>
           <p>
-            {getFailedOperatorsNames(failedOperators)} failed to install. Due to this, the cluster
-            will be degraded, but you can try to install the operator from the Operator Hub. Please
-            check the installation log for more information.
+            {getFailedOperatorsNames(failedOperators, t)} failed to install. Due to this, the
+            cluster will be degraded, but you can try to install the operator from the Operator Hub.
+            Please check the installation log for more information.
```

Comment:
missing i18n

Reply:
In ocm folder I don't add i18n translations.
Only in cim and common folder